docs/appendix.rst

```diff
@@ -200,6 +200,13 @@ This documentation uses a few special terms to refer to Python types:
       define the use of a query language, it is specified with the string
       ``DMTF:QCL``. CQL is described in DMTF standard :term:`DSP0202`.
 
+   DMTF Schema
```

Comment:
We need a general term for a schema even if it comes from a vendor and not from the DMTF. The term for that is "CIM schema". Please change to using "CIM schema".

There is a number of uses of the term "DMTF schema" that would need to be adjusted as well.

Finally, the description of this term also needs to be generalized, e.g. in "A schema in the DMTF CIM model"

Reply:
These are different entities.  the DMTF refers to their releases as CIM schemas also.  We agree that a CIM_schema is, defined by the prefix characters on any given class.  Changed all uses of the use of schema that reference the prefix to CIM Schema.  The appendix is pretty much taken from the specification itself.